data/brands/shop/books.json

```diff
@@ -656,7 +656,7 @@
       "note": "https://github.com/osmlab/name-suggestion-index/issues/5500",
       "preserveTags": ["^name"],
       "tags": {
-        "brand": "National Book Store",
+        "brand": "Norli",
```

Comment:
The ”National Book Store“ was add as brand when ”Norli“ was collected to NSI： https://github.com/osmlab/name-suggestion-index/commit/8453824e8e91ce2bd1621292316b2913b29d5b50  (Nov 3, 2020)

But it does seem strange, because after checking the information, it is indeed unrelated to the government